src/test/regress/multi_schedule

```diff
@@ -191,7 +191,8 @@ test: multi_modifications
 test: multi_distribution_metadata
 test: multi_generate_ddl_commands multi_create_shards multi_prune_shard_list multi_repair_shards
 test: multi_upsert multi_simple_queries multi_data_types
-test: multi_utilities foreign_key_to_reference_table validate_constraint
+test: multi_utilities
```

Comment:
multi_utilities              ... ok        22756 ms

This is one of the longest test file. Maybe we should consider splitting it into smaller files? Running this sequential would make the tests even longer :/

Reply:
This is fixed by #3507

Reply:
I think it'd be good to add a comment, otherwise it is likely that someone else would add a test next to it?

Reply:
done